React/Base/RCTPlatform.m

```diff
@@ -39,7 +39,11 @@ @implementation RCTPlatform
     @"forceTouchAvailable": @(RCTForceTouchAvailable()),
     @"osVersion": [device systemVersion],
     @"systemName": [device systemName],
+#if TARGET_OS_TV
+    @"interfaceIdiom": interfaceIdiom(UIUserInterfaceIdiomTV),
+#else
     @"interfaceIdiom": interfaceIdiom([device userInterfaceIdiom]),
```

Comment:
I agree, but am having difficulty finding definite Apple documentation stating that [device userInterfaceIdiom] will return this value if and only if the code is running on Apple TV.  I'll replace this with a comment, so we can more easily find the issue if Apple changes things underneath us in the future....